content/docs/start/data-management/metrics-parameters-plots.md

```diff
@@ -99,29 +104,43 @@ files to be versioned by Git.
 }
 ```
 
-You can view tracked metrics with `dvc metrics show `:
+As you can see, it's a simple json file and the structure is up to you and your
+project (see [supported formats] to learn more).
+
+DVC doesn't care how you generate [metric files][metrics file] - you can create
+them whichever way you like. As long as metrics are numerical DVC can pick them
```

Comment:
Can we try to fit it into 1-2 sentences? I mean these three paragraphs ... it's too verbose I think, a lot of boilerplate ... e.g. 

`doesn't care how you generate` pretty much the same as `you can create them whichever way you like`

`As long as metrics are numerical` - if we want to mention this, let's mention above as part of the format

visualize - I think view was better in this context tbh